README.md

```diff
@@ -1 +1,152 @@
-# school_library
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your commit messages capitalized to make your pull request professional 👍